src/js/views/eva-variant-view.js

```diff
@@ -156,143 +524,222 @@ EvaVariantView.prototype = {
-            return '';
-        }
-        var _conservedRegionTable = '<div class="row"><div class="col-md-8"><table class="table hover">'
-        _conservedRegionTable += '<tr><th>Source</th><th>Score</th></tr>'
-        _.each(_.keys(conservedRegionScores), function (key) {
-            console.log(this[key])
-            _conservedRegionTable += '<tr><td>' + this[key].source + '</td><td>' + this[key].score + '</td></tr>'
-
-        }, conservedRegionScores);
-        _conservedRegionTable += '</table></div></div>'
-
-        return _conservedRegionTable;
+    _renderConsequenceTypeData: function (variantDataArray) {
+        var _this = this;
+        var variantIndex = 1;
+        return variantDataArray.map(function(data) {
+            var consequenceTypeHeading = '<h4 class="variant-view-h4"> Consequence Types' + (data.repr ? " for "+data.repr : "") +  '</h4>';
+            var noDataAvailableSection = consequenceTypeHeading + '<div>No Consequence Type data available</div>';
+            if(_.isUndefined(data.annotation)){
+              return noDataAvailableSection;
+            }
+            var annotation = data.annotation.consequenceTypes;
+            if (!annotation) {
+                return noDataAvailableSection;
+            }
+            annotation = annotation.sort(_this._sortBy('ensemblGeneId', _this._sortBy('ensemblTranscriptId')));
+            var _consequenceTypeTable = consequenceTypeHeading + '<div class="row"><div><table id="consequence-type-summary-' + variantIndex + '" class="table hover" style="font-size: small">';
+            _consequenceTypeTable += '<thead><tr><th>Ensembl Gene ID</th><th>Ensembl Transcript ID</th><th>Accession</th><th>Name</th></tr></thead><tbody>';
+            _.each(_.keys(annotation), function (key) {
+                var annotationDetails = this[key];
+                var soTerms = this[key].soTerms;
+                _.each(_.keys(soTerms), function (key) {
+                    var link = '<a href="http://www.sequenceontology.org/miso/current_svn/term/' + this[key].soAccession + '" target="_blank">' + this[key].soAccession + '</a>';
+                    var so_term_detail = consequenceTypeDetails[soTerms[0].soName];
+                    var color = '';
+                    var impact = '';
+                    var svg = '';
+                    if (!_.isUndefined(so_term_detail)) {
+                        color = so_term_detail.color;
+                        impact = so_term_detail.impact;
+                        svg = '<svg width="20" height="10"><rect x="0" y="3" width="15" height="10" fill="' + color + '"><title>' + impact + '</title></rect></svg>';
+                    }
+
+                    var ensemblGeneId = '-';
+                    if (annotationDetails.ensemblGeneId) {
+    //                     ensemblGeneId = '<a href="http://www.ensembl.org/Homo_sapiens/Gene/Summary?g='+annotationDetails.ensemblGeneId+'" target="_blank">'+annotationDetails.ensemblGeneId+'</a>';
```

Comment:
It seems this line and one not far below were already commented. Could you please just delete them?